Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -76,6 +76,10 @@ public double translateTypicalShootingPositionToDistance(TypicalShootingPosition
         }
     }
     
+    public double getTargetSpeedForRange(double rangeInInches) {
+        return flushToBoilerTargetSpeed.get() + trimFlushToBoilerSpeed.get();
```

Comment:
If it isn't used, remove it... otherwise you're exposing an API that isn't really functional. We don't need to implement code with the hope of guessing what we'll use in the future.